packages/mux-video/src/index.ts

```diff
@@ -362,11 +362,34 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElement> implements Pa
     const video_id = this.getAttribute(Attributes.METADATA_VIDEO_ID);
     const video_title = this.getAttribute(Attributes.METADATA_VIDEO_TITLE);
     const viewer_user_id = this.getAttribute(Attributes.METADATA_VIEWER_USER_ID);
+
+    const otherMetadataAttrs: { [key: string]: string } = {};
```

Comment:
suggestion(non-blocking): Per your callout in your PR description, we shouldn't need to treat the "well-defined metadata attributes" differently here. I'd remove them so we can simplify this code.